### PR TITLE
Feature/add styling feature

### PR DESCRIPTION
**Pull request by koachcode**
## Styling Feature
1. Added responsiveness
2. Add Animation
3. Fix linter
4. I will add other styling  features later

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -55,6 +55,7 @@ h3 {
   width: 50px;
   border-radius: 100%;
   border: 2px solid rgba(0, 0, 0, 0.35);
+  margin-left: 25px;
 }
```
Comment (elyor-doniyorov) [inline src/App.css:60]: - [ ] Great job! However, I suggest paying attention to even details in order to stick to the design. For example, there are differences between the Zeplin and your design in `spacing and colors`. Take a look at the below screenshot to see 👇 
- Zeplin design 👇 
Screenshot (202)
- Your design 👇 
Screenshot (203)_LI
Kindly work on the differences and try to stick to the given design 🙏

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @Ginohmk, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (koachcode) [reply src/App.css:60]: Thank you. I will work on it . 👍

**Review by DeliceLydia — APPROVED** [review]
# Congratulations! 🎉🎉

## Hi @Ginohmk, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by koachcode** [comment]
Thank you @DeliceLydia

**Pull request merged**